src/instructlab/model/evaluate.py

```diff
@@ -465,6 +465,16 @@ def evaluate(
     tls_client_passwd,  # pylint: disable=unused-argument
     enable_serving_output,
 ):
+    # ensure user is passing full safetensors and not a GGUF if using MMLU or MMLUBranch
+    # https://github.com/instructlab/eval/issues/50
+    if model is not None:
+        if model.lower().endswith(".gguf") and benchmark in ["mmlu", "mmlu_branch"]:
```

Comment:
(Looking at the other comment about only accepting .safetensors here, this may be not important anyway but:) please use symbols instead of bare strings:

```
class Benchmark(str, enum.Enum):
    MMLU = "mmlu"
    MMLU_BRANCH = "mmlu_branch"
```

Reply:
@booxter thanks for the tip - I've just pushed up a different implementation - would value your feedback on it!